conformance/tests/gateway-unsupported-route-kind.yaml

```diff
@@ -30,5 +30,5 @@ spec:
         namespaces:
           from: All
         kinds:
-        - kind: UDPRoute
+        - kind: TCPRoute
```

Comment:
Should having a `TCPRoute` carrying HTTP traffic be allowed? What would be the use-case in using a `TCPRoute` instead of an `HTTPRoute`?

Reply:
Yeah this was more confusing than helpful, I refactored this test to more clearly be about an Invalid kind since that seemed to be the original goal. I'd been getting this one confused with the related "disallowed" kind tests.